app/src/main/cpp/skyline/common.h

```diff
@@ -338,6 +339,18 @@ namespace skyline {
         constexpr std::size_t Hash(std::string_view view) {
             return frz::elsa<frz::string>{}(frz::string(view.data(), view.size()), 0);
         }
+
+        /**
+         * @brief Fills an array with random data from a Mersenne Twister pseudo-random generator
+         * @note Currently the generator is not seeded so produced values will be identical between two runs
+         */
+        template<typename T> requires (std::is_integral_v<T>)
+        void GetRandomBytes(std::span<T> bytes) {
+            thread_local std::mt19937_64 generator{};
```

Comment:
I could be wrong but isn't brace initialization pointless for non-POD types like `mt19937_64` where the default constructor would automatically be run